script/SubmitUpgradeProposalScript.s.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// slither-disable-start reentrancy-benign
+
+pragma solidity 0.8.26;
+
+import {Script} from "forge-std/Script.sol";
+import {SharedGovernorConstants} from "script/SharedGovernorConstants.sol";
+import {GovernorUpgradeable} from "openzeppelin-upgradeable/governance/GovernorUpgradeable.sol";
+import {AccessControlUpgradeable} from "openzeppelin-upgradeable/access/AccessControlUpgradeable.sol";
+import {CreateProposalCalldata} from "script/helpers/CreateProposalCalldata.sol";
+
+contract SubmitUpgradeProposalScript is Script, SharedGovernorConstants, CreateProposalCalldata {
+  // TODO: Update `PROPOSER` to script msg.sender who will submit the proposal.
```

Comment:
Let's refactor `SubmitUpgradeProposalScript` to broadcast from the forge wallet rather than this `PROPOSER` address